etc/kayobe/grafana.yml

```diff
@@ -0,0 +1,58 @@
+---
+###############################################################################
+# Grafana configuration.
+
+# Grafana local admin user name. If you are deploying Monasca Grafana this
+# should not conflict with an OpenStack user name.
+#grafana_local_admin_user_name:
+
+# Path to git repo containing Grafana dashboards. Eg.
+# https://github.com/stackhpc/grafana-reference-dashboards.git
+#grafana_monitoring_node_dashboard_repo:
+grafana_monitoring_node_dashboard_repo:
+  "https://github.com/stackhpc/grafana-reference-dashboards.git"
+
+# Path to which Grafana dashboards will be cloned to a monitoring node
+#grafana_monitoring_node_dashboard_repo_path:
+
+# The Grafana organisation for the control plane. Note that for Monasca
+# Grafana with domain support the format is:
+# organisation_name@openstack_domain
+#grafana_control_plane_organisation:
+grafana_control_plane_organisation: "monasca@default"
+
+# A list of datasources to configure. See the stackhpc.grafana-conf role
+# for a list of supported datasources. Example:
+#
+# grafana_datasources:
+#   monasca_api:
+#     port: 8082
+#     host: monasca-api
+#   monasca_log_api:
+#     port: 5607
+#     host: monasca-log-api
+#   elasticsearch:
+#     port: 9200
+#     host: monasca-elasticsearch
+#     project_id: "some_id"
+#
+#grafana_datasources:
+grafana_datasources:
+  # TODO: Change to VIP when these are behind HAProxy
+  monasca_api:
+    port: 8082
+    host: 10.60.253.3
+  monasca_log_api:
+    port: 5607
+    host: 10.60.253.3
+  elasticsearch:
+    port: 9200
+    host: 10.60.253.3
+    project_id: "1c8ec8e9fdc64895ada3c91af6bef423"
+
+# The path, relative to the grafana_monitoring_node_dashboard_repo_path
+# containing the dashboards. Eg. /prometheus/control_plane
+#grafana_monitoring_node_dashboard_path:
+grafana_monitoring_node_dashboard_path: "/monasca/control_plane"
```

Comment:
As in kayobe review, could move this with the other variables?

Reply:
done